src/swagger-specs/policy-service.yaml

```diff
@@ -1274,85 +1423,81 @@ definitions:
           dataSetLabels:
             $ref: '#/definitions/DataSetLabels'
   MarketingActionConstraints:
-    required:
-      - timestamp
-      - clientId
-      - userId
-      - imsOrg
-      - marketingActionRef
-      - duleLabels
-      - violatedPolicies
     properties:
       timestamp:
-        description: The timestamp when this constraints response was generated
+        description: A timestamp when the response was generated.
         type: integer
         format: int64
         example: 1529699072556
       clientId:
-        description: ClientID of the client that requested these constraints
+        description: The ID of the client that requested the evaluation.
         type: string
         example: "{CLIENT_ID}"
       userId:
-        description: User Id of the user that requested these constraints
+        description: The ID the user that requested the evaluation.
         type: string
         example: "{USER_ID}"
       imsOrg:
-        description: The imsOrg on the request that generated these constraints
+        description: The ID of the IMS Org that generated the evaluation.
         type: string
         example: "{IMS_ORG}"
+      sandboxName:
+        description: The name of the sandbox that the evaluation took place in.
+        type: string
+        example: "prod"
       marketingActionRef:
-        description: Marketing action reference to which these constraints apply
+        description: The marketing action to which the evaluation applies.
         type: string
         format: uri
+      duleLabels:
+        description: >-
+          List of data usage labels either specified in request (if using the GET method) or discovered from a provided dataset ID and optional fields (if using the POST method).
```

Comment:
Instead of having this weird description where we describe two different actions, would it be better to create two different objects - one for the GET request and the other for the POST request?